### DISCUSSION TITLE
Bullets support?

### DISCUSSION THREAD

**Discussion by bfitzpat** [vkbo/novelWriter#569] [Ideas]
Markdown offers simple formatting for bulleted lists (* or -) and numbered lists (1, 2, 3), even mixed lists. These don't seem to be supported. Any chance that can be included?

**Comment by vkbo** [comment]
novelWriter is not supposed to be another markdown editor. There's a lot of good ones out there, like GhostWriter. It just uses markdown syntax for headings and emphasis. It's a step up from plain text, that's all.

The parsing currently uses a fairly simple, fast, and linear approach. It isn't well suited for defining multi-line blocks like tables and lists. I'm not sure such features are needed for writing fiction. I'm planning to add a few new features like block quotes and footnotes though.

**Reply by bfitzpat** [reply]
Totally fair. And I'm not exactly looking to use it for novel writing as much as general book construction. In my case, for a role-playing game project I'm working on. The project structure you have set up works great for me except for the lack of list support. 

Even if it just looked at bullets and knew to put them on separate lines, that would be enough. Right now it lumps them all together because of the lack of a space between them. I can just add an extra line between them for now I think.

Thanks for creating a great project!

**Reply by vkbo** [reply]
Actually, the editor supports markdown hard line breaks too, as well as renders tabs correctly, so you can just make a list with asterisks and hard line breaks.

The markdown hard line break is two (or more) spaces at the end of the line. No need to put an empty line in between.

You can also render simple table-like lists with tabs and hard line breaks. I use it a lot in my notes when listing character attributes, etc.

**Reply by bfitzpat** [reply]
That's a great little hack (two spaces at the end of the line). Thanks!

**Reply by vkbo** [reply]
I realise it would probably be useful to have the bullet character available from the insert menu. I'll add it for the 1.1 release.

**Comment by kerryland** [comment]
I just wanted to add a +1 for a bulleted list. I agree it's not needed for a novel, but it's great for lists in 'Character' and, especially, 'Plot' files.

**Reply by vkbo** [reply]
You can still make lists with hard line breaks. There just isn't automatic support for generating nested list structures. I've added various bullet characters to the Insert menu as well.
